MyOnboarding/test/MyOnboardingApp.Services.Tests/Validation/TextCheckingValidatorTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using MyOnboardingApp.Contracts.Models;
+using MyOnboardingApp.Services.Validation;
+using MyOnboardingApp.TestUtils.Extensions;
+using NUnit.Framework;
+
+namespace MyOnboardingApp.Services.Tests.Validation
+{
+    [TestFixture]
+    public class TextCheckingValidatorTests
+    {
+        private readonly TextCheckingValidator _validator = new TextCheckingValidator();
+
+
+        [Test]
+        public void Validate_ItemWithWhiteSpaceTextSpecified_ReturnsFalse()
+        {
+            var itemToTest = new TodoListItem
+            {
+                Id = new Guid("00112233-4455-6677-8899-aabbccddeeff"),
+                Text = "          ",
```

Comment:
Consider adding multiple test cases here - like line breaks, tabs and others (up to five test cases will surely be enough) 😉